Tests/Marketplace/search_and_install_packs.py

```diff
@@ -303,10 +311,19 @@ def install_packs_private(client: demisto_client,
                                  test_pack_path=test_pack_path)
 
 
+def get_error_ids(body: str) -> set[str] | None:
+    with contextlib.suppress(JSONDecodeError):
+        response_info = json.loads(body)
+        return {error["id"] for error in response_info.get("errors", [])}
+    return None
```

Comment:
Why not return an empty list? (it takes more memory, but the return value will be `set[str]`

Reply:
sure